templates/web/panels/panel.tt.html

```diff
@@ -169,7 +158,7 @@ <h3 [% IF panel_group_element.panel_group_id.defined %]id="panel_group_[% panel_
                         [% END %]
                     [% END %]
                 >
-                <div style="float:left;[% IF text_element.valign == 'middle' %]line-height:72px;[% END %]">
+                <div class="text-container[% IF text_element.valign == 'middle' %]valign-middle[% END %]">
```

Comment:
Missing a space, otherwise will generate class="text-containervalign-middle"

```suggestion
                <div class="text-container[% IF text_element.valign == 'middle' %] valign-middle[% END %]">
```